mappings/18w01a#18w02a.tinydiff

```diff
@@ -4285,16 +4285,16 @@ c	net/minecraft/unmapped/C_6680357$C_4414943		Anchor
 		p	1			entity
 	m	()Ljava/util/Map;	m_8403822		m_8403822
 	m	()V	<clinit>		<clinit>
-c	net/minecraft/unmapped/C_6893850		net/minecraft/unmapped/C_6893850
-	m	(Lnet/minecraft/unmapped/C_7278838;)I	m_9639136		m_9639136
-		p	1			p_1
-	m	(Lnet/minecraft/unmapped/C_7278838;)I	m_0243663		m_0243663
-		p	1			p_1
-	m	()I	m_4744614		m_4744614
-	m	()Lnet/minecraft/unmapped/C_7721249;	m_8032982		m_8032982
-	m	()Lnet/minecraft/unmapped/C_0241772;	m_1885058		m_1885058
-	m	()Ljava/lang/String;	m_6469872		m_6469872
-	m	()F	m_7620332		m_7620332
+c	net/minecraft/unmapped/C_6893850		net/minecraft/item/armor/IArmorMaterial
```

Comment:
From the convo on Discord I gathered that the armor materials are mostly referenced through this interface, so maybe this could be named `ArmorMaterial` and the class that defines the vanilla armor materials `ArmorMaterials` or maybe `BuiltInArmorMatetials` (there is some precedent in our mappings for the "built in" prefix I think)?